modules/aws/cassandra/locals.tf

```
@@ -22,6 +22,7 @@ locals {
     resource_type                = "t3.large"
     resource_count               = 3
     resource_root_volume_size    = 64
+    encrypt_ebs_volume           = true
```

```suggestion
    encrypt_volume           = true
```
so that we can use the same name for Azure.

---

Fixed in cf2c465